README.md

```diff
@@ -1 +1,76 @@
-# haaukins-api
\ No newline at end of file
+# Haaukins API
+
+Haaukins API allows the user to connect directly Kali Linux VM avoiding normal Haaukins steps. Unlike Haaukins, in which
+you need to have an event up and running in order to sign up and connect Kali Linux VM, on the API you just need to select
+the challenges you want to have in your Environment and run it. In almost a minute the environment will be ready and the 
+user will be automatically redirect to Guacamole that render the Kali linux VM.
```

Comment:
redirected to Guacamole which creates RDP connection through browser to Kali Linux VM.